### PR TITLE
Enhanced the banner design on the website.

### PR DESCRIPTION
**Pull request by cankocman**
Screenshot from 2020-11-02 17-29-22
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by proful** [comment]
@sarathkumar6 @vasudeveloper001 @JoelVinayKumar

This banner design is bit different. You guys thought?

**Comment by JoelVinayKumar** [comment]
Not my changes

**Comment by vasudeveloper001** [comment]
No. It's something he came up with himself.

**Comment by cankocman** [comment]
Which changes are you talking about? I didn't see them.

**Comment by cankocman** [comment]
Which changes are you talking about? I didn't see them.

@JoelVinayKumar @vasudeveloper001

**Comment by vasudeveloper001** [comment]
The header redesign of centring the logo and navigation. Let's wait and see if others like it. 

I don't think this is the banner design that we were talking. That's all. @proful can elaborate more on what he meant by banner design enhancement here.

**Comment by cankocman** [comment]
Aince there are only 3 elements in navbar i thoughy centering both logo and ul would be good. Because i am using a 27" Monitor and on my monitor there was a lot of gap between logo and li elements. Also i didn't see any instructions to follow. Is there any? 
@JoelVinayKumar @vasudeveloper001

**Comment by JoelVinayKumar** [comment]
Are we doing tabbed navigation ? Or vertical page navigation ?

**Comment by proful** [comment]
I think this design is not aligned with @JoelVinayKumar and @vasu001 so we can't approve and merge the changes.

@cankocman please talk to Joel and Saurabh they should be able to give you task to work on

File: twindle-web/index.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
   <body>
     <!-- Header Section Code for designing navigation -->
     <header>
-      <a href="/"><img src="./images/logo.svg" alt="Twitter Logo" /></a>
+      <a href="/" id="logo"><img src="./images/logo.svg" alt="Twitter Logo" /></a>
+    </header>
```
Comment (RafaelBatman55) [inline twindle-web/index.html:13]: You can't divide your header in two elements. Please, keep just one as original document.

Comment (cankocman) [reply twindle-web/index.html:13]: Thanks. I missed that part my bad.

**Comment by cankocman** [comment]
@vasudeveloper001 @JoelVinayKumar Ok, if you can explain what do you actually want i can work on it.

**Comment by vasudeveloper001** [comment]
@cankocman I think no centering on header is fine. That's one. Responsiveness can be done as that is also on the table. 
 By Banner, I it has something to do with enhancing the main area of page. You can discuss more with @JoelVinayKumar and collaborate with him on that front. 

First discuss with @JoelVinayKumar now before working on the code @cankocman

**Pull request closed without merging**